### PR TITLE
Fix typos and improve formulations in FAQ

### PR DESCRIPTION
**Pull request by cbjornram**
Hi,

I started doing some typo fixes and ended up cleaning up a bit more. Please note that I removed the entire **"Why is MAVSDK written in C++?"** question and answer due to the very similar question and answer **"Why was the core of the SDK written in C++?"**.

Please make changes if you see things you don't agree with, or just let me know and I will do it instead.

All the best,
Christoffer

### CODE REVIEW

File: en/faq.md [hunk 1]
```diff
@@ -9,8 +9,8 @@ Additionally, the library needs to be lightweight and fast, so it does not slow
 
 Yes.
 - The MAVSDK C++ library allows C++ applications to connect to multiple vehicles at a time.
-- Python, Swift, and other language wrappers an only connect to a _single vehicle at a time_.
-  However you can intantiate multiple copies of wrappers in order to connect to multiple systems.
+- Python, Swift, and other language wrappers can only connect to a _single vehicle at a time_.
+  However you can initiate multiple copies of wrappers in order to connect to multiple systems.
```
Comment (hamishwillee) [inline en/faq.md:13]: intantiate is correct here.

File: en/faq.md [hunk 2]
```diff
@@ -64,36 +64,30 @@ Check out [C++/Contributing/Plugins](cpp/contributing/plugins.md).
...
 - Essentially, it would mean that all MAVLink APIs would have to be duplicated by the [proto APIs](https://github.com/mavlink/MAVSDK-Proto/tree/main/protos).
   This would increase the API and code size considerably.
 - Alternatively, the API could be exposed without types, using some sort of runtime access like `get_message("ATTITUDE").get_field("pitch").as_float()`.
   This would come with the cost of no type safety and runtime overhead for parsing the strings.
 
 From a MAVSDK project point of view, there is also an advantage of not having a passthrough available in language wrappers; it encourages that required features are contributed back to the open-source project, rather than implemented in private using passthrough, and thus benefitting everyone.
 
-### Why is MAVSDK written in C++?
-
-The requirements for MAVSDK were that it can efficiently run in embedded setups, e.g. as part of an app or onboard a drone on the companion computer which suggested C or C++.
```
Comment (hamishwillee) [inline en/faq.md:94]: Yes, it is good to delete the heading. But I would merge the content into the other topic.

**Comment by hamishwillee** [comment]
Thanks @cbjornram - yes, most of this is much better. I added a couple of comments. @julianoes or @JonasVautherin might have more to say.

**Comment by cbjornram** [comment]
Thank you for your feedback @hamishwillee. I have made a commit with your suggestions/corrections and am waiting if anyone else sees anything more.

**Comment by hamishwillee** [comment]
Cool. Ping me again if no one replies by mid next week.

**Review by JonasVautherin — APPROVED** [review]
Nice!

**Comment by hamishwillee** [comment]
Thanks @cbjornram - really good to fix these things as we see them.

**Pull request merged**